packages/repository/README.md

```diff
@@ -168,7 +168,13 @@ coercion. The following types are supported out of box.
 - ArrayType
 - UnionType
 
-# Use Repository
+## Installation
+
+```
+$ npm install --save @loopback/repository
+```
+
+## Basic use
```

Comment:
That makes sense to me, especially that we declared that we are experimenting and might refactor things out in the first lines:
```
NOTE: This module is experimental and evolving. It is likely going to be refactored and decomposed into multiple modules
```
@bajtos thoughts?